optuna/trial/__init__.py

```diff
@@ -4,3 +4,13 @@
 from optuna.trial._frozen import FrozenTrial  # NOQA
 from optuna.trial._state import TrialState  # NOQA
 from optuna.trial._trial import Trial  # NOQA
+
```

Comment:
`# NOQA`s can be removed samely as `optuna/__init__.py`.

Reply:
Oops! I removed it in https://github.com/optuna/optuna/pull/2013/commits/671748794c76e1a12c7d330f20e323b98d71c702.